app/controllers/external_resources_controller.rb

```diff
@@ -59,8 +59,17 @@ def set_external_resource
 
     # Only allow a list of trusted parameters through.
     def external_resource_params
-      params.require(:external_resource).permit(:name, :website_url, :facebook_url,
-                                                :phone, :description, :display_on_website,
-                                                :youtube_identifier, :location_id, tags: [])
+      params.require(:external_resource).permit(
+          :description,
+          :display_on_website_end,
+          :display_on_website_start,
```

Comment:
Converting display_on_website from boolean to date range. (I know postgres supports date range fields, but leaving them separate fields allows the user to leave these blank and therefore not part of what hides/shows things on the website.)